qa/rpc-tests/test_framework/connectrum/constants.py

```diff
@@ -0,0 +1,34 @@
+
+# copied values from electrum source
+
+# IDK, maybe?
+ELECTRUM_VERSION = '2.6.4'  # version of the client package
+PROTOCOL_VERSION = '0.10'   # protocol version requested
```

Comment:
Isn't this ancient?  I think we are on protocol version 1.4

Reply:
Yes, but this isn't even used. There is a lot we can remove if we want to modify this to keep it to a minimal. For example the bootstrap servers below, all the `if __name__ == '__main__':` clauses and TOR onion stuff